packages/manager/apps/metrics/package.json

```diff
@@ -59,7 +62,9 @@
     "jquery": "^2.1.3",
     "moment": "^2.19",
     "moment-duration-format": "^1.3",
-    "ovh-api-services": "^11.0.1"
+    "oclazyload": "^1.1.0",
+    "ovh-api-services": "^11.0.1",
+    "whatwg-fetch": "^3.5.0"
```

Comment:
If you pay attention, this new version is added into the `yarn.lock` file.
Maybe we can rely on a single version (up-to-date).

Could be applied across all following packages then:

```sh
$ git grep "whatwg-fetch" -- "*.json"
packages/manager/apps/cloud-connect/package.json:    "whatwg-fetch": "^3.0.0"
packages/manager/apps/cloud/package.json:    "whatwg-fetch": "^3.0.0",
packages/manager/apps/dbaas-logs/package.json:    "whatwg-fetch": "^3.0.0"
packages/manager/apps/dedicated/package.json:    "whatwg-fetch": "^3.0.0"
packages/manager/apps/hub/package.json:    "whatwg-fetch": "^3.0.0"
packages/manager/apps/public-cloud/package.json:    "whatwg-fetch": "^3.0.0"
packages/manager/apps/telecom/package.json:    "whatwg-fetch": "^3.0.0"
packages/manager/apps/web/package.json:    "whatwg-fetch": "^3.0.0"
packages/manager/modules/config/package.json:    "whatwg-fetch": "^3.0.0"
packages/manager/modules/ufrontend/package.json:    "whatwg-fetch": "^3.0.0"
```

Is that make sense?

Reply:
Done for `develop`, https://github.com/ovh/manager/pull/4281 will rebase

Reply:
Rebase has been done